source/develop/release-management/features/storage/incremental-backup.html.md

```diff
@@ -12,173 +12,421 @@ feature_status: Design
 
 ## Summary
 
-Incremental backup allows a user to backup only the changed blocks in
-all the disks belonging to a VM snapshot. The typical use case is backing
-up the current changes in the active snapshot. Another use case may be
-backing up the previous snapshot if a user added a snapshot since the
-last backup.
+Incremental backup allows a user to backup only the changed blocks since
+the last backup, instead of performing a full backup on every backup.
+If the VM is running during the backup, new data written after the
+backup is started is not included in the backup.
 
-Note: users using RAW files will have to create a snapshot on top to
-allow incremental backup to start.
+## Owner
 
+- Nir Soffer (<nsoffer@redhat.com>)
+- Daniel Erez (<derez@redhat.com>)
 
-## Owner
 
-*   Nir Soffer (<nsoffer@redhat.com>)
-*   Daniel Erez (<derez@redhat.com>)
+## High level Design
+
+### Creating VM
+
+When adding a disk, the user should enable incremental backup for every
+disk. If incremental backup is enabled for a disk, a backup application
+can included the disk during incremental backups.
```

Comment:
User can enable incremental backup for any disk to enable incremental backups for the disk. Disk not marked for incremental backup or using raw format can still be fully backed using the new APIs without the need to create and delete snapshots.